packages/bridge-ui/src/eth/bridge.spec.ts

```diff
@@ -160,6 +161,8 @@ describe("bridge tests", () => {
         message: {
           owner: "0x",
           srcChainId: BigNumber.from(167001),
+          destChainId: BigNumber.from(31336),
```

Comment:
we should probably use a constant here now.

Reply:
this is just a test file

Reply:
i still think that TAIKO_CHAIN_ID etc. is slightly more semantic, but yeah not a big deal anyways.